signals.mss

```diff
@@ -240,6 +240,33 @@ Format details:
     marker-allow-overlap: true;
   }
 
+  /*********************************************/
+  /* DE wrong road signal Zs 6(DB) /Zs 7(DR)  */
+  /*********************************************/
+  /* sign */
+  [zoom>=17]["wrong_road"="DE-ESO:zs6"]["wrong_road_form"="sign"] {
```

Comment:
`DE-ESO:db:zs6` and `DE-ESO:dr:zs7` are suggested by the documentation and used by the JOSM preset but `DE-ESO:zs6` is not. That's why I suggest to change it to `DE-ESO:db:zs6` and skip the DR version of the sign because apart from 4 signs all Zs 6 signs are DB-like. The four remaining signs are mounted at Hl signals in Reuth (bei Plauen/Vogtland), Pölchow and Schwaan, i.e. their lifetime is limited.

https://taginfo.geofabrik.de/europe/germany/keys/railway:signal:wrong_road#values

See also https://github.com/OpenRailwayMap/OpenRailwayMap/pull/800#discussion_r1002692131